src/js/components/toc.coffee

```diff
@@ -6,13 +6,17 @@ React = require 'react'
 
 Toc = React.createClass
   render: ->
-    activity = @props.activity
-    return (div {}) unless activity?
+    sequence = @props.sequence
+    return (div {}) unless sequence?
     pageStudents = getPageStudents @props.students
     # In the future, we can replace [activity] with sequence.activities.
-    activityStudents = getActivityStudents [activity], pageStudents
-    (div {className: "TOC"},
-      (div {className: "activity-wrap"}, 
+    acts = _.map sequence.activities, (activity) =>
+      activityStudents = getActivityStudents [activity], pageStudents
```

Comment:
I think my initial intention was to call this method like:
`activityStudents = getActivityStudents sequence.activity, pageStudents` 
one level higher, just once, as it returns hash of results organized by activity ID. But it works like this too. ;)